### PR TITLE
Record host checkin times

### LINKED ISSUE #19: record host checkin times in database
We should add a facet for hosts and save the time they checked in to Foreman.

### PR DESCRIPTION
**Pull request by timogoebel**
Fixes #19.

### CODE REVIEW

File: db/migrate/20180711111903_create_dlm_facets.foreman_dlm.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateDlmFacets < ActiveRecord::Migration[5.1]
+  def change
+    create_table :dlm_facets do |t|
+      t.references :host, null: false, foreign_key: true, index: true, unique: true
+      t.column :last_checkin, :datetime
```
Comment (leewaa) [inline db/migrate/20180711111903_create_dlm_facets.foreman_dlm.rb]: I would call this one `last_checkin_at`

**Comment by timogoebel** [comment]
@leewaa: Updated and added a controller test for the host API.

**Comment by leewaa** [comment]
@timogoebel please rebase

**Comment by timogoebel** [comment]
@leewaa: Done, can you do a final review and merge when travis is 🍏?

**Comment by leewaa** [comment]
Sure

**Comment by leewaa** [comment]
Good to go.

**Pull request merged**